tcp/tcp.py

```diff
@@ -99,8 +99,10 @@ def __process_requests(self, client_socket, client_addr):
         This function serves the control socket's coming requests.
         """
         reinstall_file = list()
-        with open('reinstall.txt') as fp:
+        with open('reinstall.txt', "r+") as fp:
```

Comment:
can you also add a try: here to handle the case where reinstall.txt doesn't exist? also please make 'reinstall.txt' a variable in piman that you import so that it is consistent here and in piman.py

Reply:
Thank you. To handle the filenotfound error when reinstall.txt does not exist, TCP will just pass silently since it would assume if reinstall.txt does not exist then no pis need to reinstall. The reinstall function on piman.py will create the reinstall file if none are present.

Reply:
did you test that? i believe open will throw an exception.